### DISCUSSION TITLE
latest news seems quite old for a random ticker

### DISCUSSION THREAD

**Discussion by hinxx** [OpenBB-finance/OpenBB#255] [Q&A] [answered]
First off all: just started using this **great** package, lots of things to still figure out and I hope to provide constructive feedback and contribute to my best abilities.

On a random use I tried to get the news of the LAZR ticker (could be any other I guess, did not try) today and noticed the list of news I got is quite outdated:

```
(✨) (dd)> news
-> Why Apple entering the EV market is 'a good thing'
https://finance.yahoo.com/news/why-apple-entering-the-ev-market-is-a-good-thing-113808559.html

-> Electric vehicle company Canoo lists on Nasdaq
https://finance.yahoo.com/video/electric-vehicle-company-canoo-lists-192550229.html

-> Lordstown Motors CEO: We have pre-sold 100,000 electric pickup trucks
https://finance.yahoo.com/news/lordestown-motors-ceo-we-have-pre-sold-100000-electric-pickup-trucks-174316742.html

-> Apple Moves Forward With Plan for Self-Driving Vehicles
https://www.investopedia.com/apple-moves-forward-with-plan-for-self-driving-vehicles-5093423?utm_campaign=quote-yahoo&utm_source=yahoo&utm_medium=referral

-> Luminar stock plunges after news that Mobileye plans to make its own lidar sensors
https://www.marketwatch.com/story/luminar-stock-plunges-after-news-that-mobileye-plans-to-make-its-own-lidar-sensors-11608067059?siteid=yhoof2
```

If I usderstood the features WIKI page correctly, the news is taken off finviz? Their web site shows more recent articles (i.e. the one from Mar 18 '21). What gives?

**Comment by DidierRLopes** [comment, marked as answer]
Thank you! I always appreciate good positive feedback. Just had a look at that particular ticker, and you're right. I assumed that the news order was correct (must have seen 2/3 examples where it seemed correct). But something seems to be off.

If you try: `news -n 30`you'll see that some of the recent articles are there. Obviously, it doesn't make any sense to not display this by order, and having their date should be a must.. I'll try to fix this soon, thank you :)

**Reply by hinxx** [reply]
No worries. Thanks for looking into it.
While you're at it, it might be super slick to have the date printed next to the news article title/URL..

**Reply by DidierRLopes** [reply]
Yea, you're right. When I was looking at it now, I was thinking "what was I thinking?" to not have a date ahah

But the thing is that from what I can see, Finviz doesn't provide date. I'll try to scrap it from the link, or get news from a different source

**Reply by hinxx** [reply]
Hmm, finviz really likes YF, at least for this ticker it seems.

The date / time info in front of the link on finviz may just have time for subsequent news during a day. :face_with_head_bandage: for scraping.

OTOH, if you're hunting for new source, maybe look at the marketwatch. The 'collection__elements j-scrollElement' divs hold all the date/time info, come from variety of sources and the list holds way more articles on the same ticker than finviz does.

**Reply by DidierRLopes** [reply]
What you think about this:

<img width="770" alt="Captura de ecrã 2021-03-22, às 22 47 42" src="https://user-images.githubusercontent.com/25267873/112067847-abc70400-8b60-11eb-9b58-53f0dfe388a1.png">

It's from here: https://newsapi.org/docs/endpoints/everything looks pretty good, no?

**Reply by DidierRLopes** [reply]
I went ahead and made the changes, see #263

**Reply by hinxx** [reply]
You're amazing! :1st_place_medal: 
PS. Love their example query `q=gamestop +stonks -sell` :hodl

**Reply by DidierRLopes** [reply]
Let's goooo! eheh
